webui/src/lib/api/index.js

```diff
@@ -853,6 +873,11 @@ class Retention {
         return response.json();
     }
 
+    async setGCPolicyPreflight(repoID) {
+        const response = await apiRequest(`/repositories/${encodeURIComponent(repoID)}/gc/rules/set_allowed`);
+        return response.status === 204;
```

Comment:
I am not sure why this is coded differently from setGCPolicy, and the logic is different from uploadPrefight.